bootcd/rpms/genesis_scripts/genesis_scripts.spec

```diff
@@ -11,6 +11,9 @@ Source2:        src/sysconfig-init.diff
 Source3:        src/tty.conf.override
 Source4:        src/genesis-bootloader
 Source5:        src/login-shell
+Source6:        src/init.d-genesis
+Source7:        src/run-genesis-bootloader
+Source8:        src/sysconfig-genesis
```

Comment:
Genesis.sysconfig is a better name, to line up with patterns in other rpms. (Nitpick)